packages/manager/src/features/Firewalls/components/DefaultFirewallChip.tsx

```diff
@@ -0,0 +1,22 @@
+import { Chip, Tooltip } from '@linode/ui';
+import React from 'react';
+
+import type { SxProps } from '@mui/material';
+
+interface Props {
+  chipProps?: {
+    sx?: SxProps;
+  };
```

Comment:
Could we just use `ChipProps` here for maintainability / extendability?

```suggestion
  chipProps?: Partial<ChipProps>;
```